sar/README.md

```diff
@@ -212,6 +215,7 @@ None of the fields is required.
 | proj:epsg                 | (✓)  | ✓ 1.7.9 | `CoordinateReferenceSystem` | **REQUIRED for *Prod*.** See comment below*. |
 | proj:wkt2 / proj:projjson | (✓)  | ✓ 1.7.9 | `CoordinateReferenceSystem` | **REQUIRED for *Prod* if `proj:epsg` is `null`.** See comment below*. |
 | proj:bbox                 | (✓)  | ✓ 1.7.5 | `ProductBoundingBox`        | **REQUIRED for *Prod* if the native CRS is not WGS84.** This is the bounding box in the native CRS of the product. Can be omitted if the native CRS is WGS84. |
+| proj:transform            |  ✗   | (✓)     | *n/a*                       | Recommended to include the affine transformation coefficients for the default grid. |
```

Comment:
Why has this been added?

Reply:
I've had the background of how this was added to the Projection extension in the first place in mind, which was also mentioned [here](https://medium.com/radiant-earth-insights/stac-1-0-beta-1-released-c7abed779d5d) in the frst paragraph of "Additional Flexibility". In the framework of COPA we're developing a prototype Sentinel-1 NRB product. From the user perspective there's a lot of potential in using packages such as [stackstac](https://github.com/gjoseph92/stackstac) to quickly/lazily load data into Xarray DataArrays. That's why I thought it makes sense to include this field as a recommendation.

Reply:
Sounds reasonable, thanks for the explanation.